components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt.ui/src/main/resources/web/workflow-mgt/template-wf-wizard.jsp

```diff
@@ -336,8 +337,8 @@
                                             selected = "selected" ;
                                         }
                                 %>
-                                    <option value="<%=templateTmp.getTemplateId()%>" <%=selected%>>
-                                        <%=templateTmp.getName()%>
+                                    <option value="<%=Encode.forHtmlAttribute(templateTmp.getTemplateId())%>" <%=Encode.forHtmlAttribute(selected)%>>
```

Comment:
No need to encode variable selected. It contains a hard coded value that denotes the selected attribute